src/indra_cogex/apps/templates/base_w_chat.html

```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %}
+
+{% block styles %}
+    {{ super() }}
+    <!-- Chat app css -->
+    <link rel="stylesheet" href="{{ url_for('static', filename='chat/css/app.css') }}">
+{% endblock %}
+
+{% block main %}
+    {{ super() }}
+    <div class="chatbubble">
+        <div class="unexpanded">
+            <div class="title">Chat with INDRA CoGEx</div>
+        </div>
+        <div class="expanded chat-window">
+
+            <div class="login-screen container">
+                <p>Log in to chat with INDRA CoGEx.</p>
+                <form id="loginScreenForm">
+                    <div class="form-group">
+                        <input type="text" class="form-control" id="fullname" placeholder="Name*" required>
+                    </div>
+                    <div class="form-group">
+                        <input type="email" class="form-control" id="email" placeholder="Email Address*" required>
+                    </div>
+                    <button type="submit" class="btn btn-block btn-primary">Start Chat</button>
+                </form>
+            </div>
+
+            <div class="chats">
+                <div class="loader-wrapper">
+                    <div class="loader">
+                        <span>{</span><span>}</span>
+                    </div>
+                </div>
+                <ul class="messages clearfix">
+                </ul>
+                <div class="input">
+                    <form class="form-inline" id="messageSupport">
+                        <div class="form-group">
+                            <input type="text" autocomplete="off" class="form-control" id="newMessage"
+                                   placeholder="Enter Message">
+                        </div>
+                        <button type="submit" class="btn btn-primary">Send</button>
+                    </form>
+                </div>
+            </div>
+
+        </div>
+    </div>
+{% endblock %}
+
+{% block scripts %}
+    {{ super() }}
+    <!-- Chat app setup -->
+    <script src="https://js.pusher.com/4.0/pusher.min.js"></script>
+    <script>
+        const pusher = new Pusher('{{ pusher_app_key }}', {
+            authEndpoint: '{{ url_for('home.pusher_authentication') }}',
+            cluster: 'mt1',
+            encrypted: true
+        });
+    </script>
+    <script type="text/javascript" src="{{ url_for('static', filename='chat/js/popper.js') }}"></script>
```

Comment:
does bootstrap bundle popper?

Reply:
Looks like it: https://bootstrap-flask.readthedocs.io/en/stable/api/#module-flask_bootstrap

Reply:
Resolved in https://github.com/bgyori/indra_cogex/pull/94/commits/58934137067b3e8be13108e609308742daed7933